cmd/kots/cli/install.go

```diff
@@ -426,6 +429,7 @@ func InstallCmd() *cobra.Command {
 				case err := <-errChan:
 					if err != nil {
 						log.Error(err)
+						// TODO: Why is this a negative exit code?
```

Comment:
Good question... I think this end up resulting in a 255 exit code:
```
$ echo $?
255
```

Reply:
let's fix this and set it to 255